specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefits.json

```diff
@@ -0,0 +1,1002 @@
+      "type": "object",
+      "properties": {
+        "armSkuName": {
+          "description": "ARM SKU name. For example, Standard_D1_V2 for reservations and ComputeSavingsPlan for savings plan.",
+          "type": "string",
+          "readOnly": true
+        },
+        "benefitId": {
+          "description": "The benefit ID is the identifier of a benefit within a benefit order.",
+          "type": "string",
+          "readOnly": true
+        },
+        "benefitOrderId": {
+          "description": "The benefit order ID is the identifier for a benefit purchase. Each benefit order ID represents a single purchase transaction. For example: A reservation order contains reservations. The reservation order specifies the VM size and region for the reservations.",
+          "type": "string",
+          "readOnly": true
+        },
+        "benefitType": {
+          "description": "The benefit type. For example, reservation or savings plan",
+          "type": "string",
+          "readOnly": true
+        },
+        "usageDate": {
+          "description": "Date corresponding to the utilization record. If the grain of data is monthly, it will be first day of month.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "IncludedQuantityUtilizationSummary": {
+      "description": "Included Quantity utilization summary resource.",
+      "x-ms-discriminator-value": "IncludedQuantity",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummary"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IncludedQuantityUtilizationSummaryProperties",
+          "title": "Benefit utilization summary for benefit kind Included Quantity."
+        }
+      }
+    },
+    "IncludedQuantityUtilizationSummaryProperties": {
+      "type": "object",
+      "description": "Included Quantity utilization summary properties.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummaryProperties"
+        }
+      ],
+      "properties": {
+        "utilizationPercentage": {
+          "description": "This is the utilized percentage for the benefit ID.",
+          "type": "number",
+          "format": "decimal",
+          "readOnly": true
+        }
+      }
+    },
+    "BenefitUtilizationResource": {
+      "description": "The Resource model definition.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      }
```

Comment:
These should really all be inherited from the common-types definitions like ProxyResource somehow. Sorry if I give the same feedback multiple times but too many PRs.

Reply:
Done.